sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurations.cs

```diff
@@ -98,6 +104,12 @@ public class TestConfigurations
         public static TenantConfiguration DefaultTargetOAuthTenant =>
             GetTenant("TargetOAuthTenant", s_configurations.Value.TargetOAuthTenantName);
 
+        /// <summary>
+        /// Gets a tenant to use for any tests that require hierarchical namespace
+        /// </summary>
+        public static TenantConfiguration DefaultNamespaceEnabledTenant =>
+            GetTenant("TargetHierarchicalNamespaceTenant", s_configurations.Value.TargetHierarchicalNamespaceTenantName);
```

Comment:
Should we be using TargetHierarchicalNamespaceTenantName instead of  hardcoded value?

Reply:
Maybe, I was just following the pattern of the TenantConfigurations in the previous lines.